README.md

```diff
@@ -1 +1,38 @@
-# boomscience
\ No newline at end of file
+# BoomScience
+BoomScience is a steampunk metroidvania platformer set in an anthropomorphic universe. The main character is a scientist that had his research on sustainable energy stolen by a company and used for bellic/war purposes against his will. His objective is to retrieve his research and prevent caos.
+## Installation
+This game is developed using Godot Engine. Install the latest version on https://godotengine.org/download, or visit http://store.steampowered.com/app/404790/Godot_Engine/.
+## Usage
+On Godot Engine Project Manager, import engine.cfg and run.
+## Contributing
+1. Fork it!
+2. Create your feature branch: `git checkout -b my-new-feature`
+3. Commit your changes: `git commit -am 'Add some feature'`
+4. Push to the branch: `git push origin my-new-feature`
+5. Submit a pull request :D
+## History
+This project was created on February 2016. A prototype was made on its first semester; the main development started on the second semester.
+A demo was released on March 24th, 2017.
+## Credits
+### Programming
+* Victor Domiciano Mendonça
+* Willian Akira Mizutani
+* Wilson Kazuo Mizutani
+* André Victor dos Santos Nakazawa
+* Isabela Blucher
+* André Ferrari Moukarzel
+* Luiz Felipe Moumdjian Girotto
+* Vitor Kei Taira Tamada
+* Gabriel Oliveira
+* Ricardo Lira da Fonseca
+### Visual Art
+* Willian Akira Mizutani
+* André Victor dos Santos Nakazawa
+* Victor Domiciano Mendonça
+* Renato Yoshio Eguti
+### Music and Sound
+* Lívia Maki Yoshikawa
+* Eduardo Silva Araújo
+## License
+Licensed under the MIT License.
+
```

Comment:
We need a License for the assets/script/gameplay. Usually we use the latest Creative Commons BY-SA (optionally with NC)